x-pack/plugins/actions/server/lib/action_executor.test.ts

```diff
@@ -23,6 +23,7 @@ import { securityMock } from '@kbn/security-plugin/server/mocks';
 import { finished } from 'stream/promises';
 import { PassThrough } from 'stream';
 import { SecurityConnectorFeatureId } from '../../common';
+import { TaskErrorSource } from '@kbn/task-manager-plugin/common';
```

Comment:
Can we add a test for what is returned when the `actionType.executor` doesn't throw an error but returns a result with an error?

Reply:
done